src/RadioButton/RadioButton.tsx

```diff
@@ -0,0 +1,79 @@
+import React, {FC, ChangeEvent} from 'react';
+import styled from 'styled-components';
+
+import {Box} from '../Box';
+
+import {theme} from '../theme';
+
+interface IFakeInput {
+  checked?: boolean;
+}
+
+type RadioButtonProps = {
+  id: string;
+  label: string;
+  checked?: boolean;
+  value: string;
+  type?: string;
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+};
+
+export const RadioButton: FC<RadioButtonProps> = ({
+  id,
+  type = 'radio',
+  label,
+  checked,
+  value,
+  onChange,
+}) => (
+  <Box flex alignItems="center" mb="24px">
```

Comment:
forever abstract design question:

setting pre-defined spacing styles such (`margin`s) vs keeping the outer box plain and unaware

Reply:
Probably a good shout to keep it generic.